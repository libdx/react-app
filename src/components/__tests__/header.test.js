import React from 'react'
import { shallow } from 'enzyme'
import Header from '../header'

describe('Header', () => {

    const brand = 'Brand'
    let component

    beforeEach(() => {
        component = shallow(<Header brand={brand} />)
    })

    it('has brand label', () => {
        const label = component.find('.brand-label') 

        expect(label.exists()).toBe(true)
    })
})
