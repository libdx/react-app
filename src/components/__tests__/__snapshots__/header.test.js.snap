// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="header"
>
  <p
    className="brand-label"
  >
    Brand
  </p>
  <p
    className="title"
  >
    Find Your Movie
  </p>
</div>
`;
